app/controllers/films_controller.rb

```diff
@@ -51,6 +54,18 @@ def destroy
 
   private
 
+  def omdb_search
+    return if params.values_at(:search, :year).include?(nil)
+
+    @film_search = FilmDecorator.new(OmdbService.new(params[:search], params[:year]))
```

Comment:
you are wrapping instance of ombd service in decorator. hmm, why like that? I would wrap client response

Reply:
Also why used you `FilmDecorator` here? I think `Film` is different kind of entity than your API response